flask.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Flask
+layout: 2017/sheet
+category: Flask  # could be split later on
```

Comment:
this category doesn't exist in `_data/categories.yml`, maybe start with a `Python libraries` category. Btw, I can help on this stuff later!